src/byrow.jl

```diff
@@ -28,18 +28,22 @@ byrow_replace(x) = x
 
 function byrow_find_newcols(e::Expr, newcol_decl)
     if e.head == :macrocall && e.args[1] == Symbol("@newcol")
-        ea = e.args[2]
+        if VERSION < v"0.7-"
```

Comment:
Also since this is in a function (but in this case the conditional is less easily separable, the condition should have an `@static` to evaluate the branch once at parse time.